app_dart/lib/src/request_handlers/check_flaky_builders.dart

```diff
@@ -102,13 +112,13 @@ class CheckFlakyBuilders extends ApiRequestHandler<Body> {
   }
 
   /// Gets the builders that match conditions:
-  /// 1. There is no [kCiYamlTargetIgnoreFlakiness] specified.
+  /// 1. The builder's ignoreFlakiness is false.
   /// 2. The builder is flaky
   /// 3. The builder is not in [ignoredBuilders].
   /// 4. The flaky issue of the builder is closed if there is one.
   /// 5. Does not have any existing pr against the builder.
   Future<List<_BuilderInfo>> _getEligibleFlakyBuilders(GithubService gitHub, RepositorySlug slug,
-      {required String content}) async {
+      {required String content, required CiYaml ciYaml}) async {
     final YamlMap ci = loadYaml(content) as YamlMap;
     final YamlList targets = ci[kCiYamlTargetsKey] as YamlList;
     final List<YamlMap?> flakyTargets = targets
```

Comment:
Some duplication here as well. Can we consolidate `ci` and `ciYaml`? Same in other APIs.

Reply:
this is needed because of other things in the code. I tried doing it the ideal way we want, but hit some problems that require their own change.